pkg/model/file/marker.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package file
+
+import (
+	"fmt"
+	"path/filepath"
+)
+
+const prefix = "+kubebuilder:scaffold:"
+
+var commentsByExt = map[string]string{
+	// TODO(v3): machine-readable comments should not have spaces by Go convention. However,
```

Comment:
Can you create an issue to track everything related to `TODO(v3)`?

Reply:
#1382, #1383, and #1384 raised. Could you add them to the corresponding milestone?